packages/renderer/src/lib/node/NodeDetails.spec.ts

```diff
@@ -18,13 +18,15 @@
 import '@testing-library/jest-dom/vitest';
 
 import type { CoreV1Event, KubernetesObject, V1Node } from '@kubernetes/client-node';
-import { render, screen, waitFor } from '@testing-library/svelte';
+import { render, screen } from '@testing-library/svelte';
 import { writable } from 'svelte/store';
 import { router } from 'tinro';
-import { beforeAll, expect, test, vi } from 'vitest';
+import { beforeEach, describe, expect, test, vi } from 'vitest';
 
-import * as kubeContextStore from '/@/stores/kubernetes-contexts-state';
+import * as states from '/@/stores/kubernetes-contexts-state';
```

Comment:
same question there

Reply:
I don't how to write this without wildcard import:

```
      vi.mocked(states).kubernetesCurrentContextNodes = nodesStore;
```